l10n_br_account/models/account_fiscal_position.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2009 - TODAY Renato Lima - Akretion
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, fields, api
+
+from .account_fiscal_position_abstract import (
+    AccountFiscalPositionAbstract,
+    AccountFiscalPositionTaxAbstract
+)
+
+
+class AccountFiscalPosition(AccountFiscalPositionAbstract,
```

Comment:
Esta tela não esta funcionando corretamente, não foi possível inserir dados no campo tax_ids e o teste automatizado esta viciado, pois não passa pelo método map_tax que utiliza os campos deste modelo.

Reply:
@mileo aqui tem um problema que vi com o @renatonlima tanto o modulo l10n_br_account_product quanto o l10n_br_account_service vão usar esse método mas não é usado aqui ( ou podemos ver uma forma de chama-lo ) se for removido daqui vai ser preciso duplicar o código nesses módulos

Reply:
@mileo para resolver este questão eu previso pegar o onchange_fiscal() que esta no módulo l10n_br_account_product e coloca-lo no l10n_br_account. para eu devo fazer esta mudança depois deste merge para não ter que fazer o rebase na migração l10n_br_account_product, pois vai dar muito conflito fazer isso.